libavfilter/dnn_filter_common.c

```diff
@@ -17,6 +17,36 @@
  */
 
 #include "dnn_filter_common.h"
+#include "libavutil/avstring.h"
+
+static char **separate_expression(DnnContext *ctx, const char *expr, const char *val_sep, int *separated_nb)
```

Comment:
ctx is not used?

Reply:
yes, if i do not use av_log here, the ctx is no longer necessary.